app/assets/stylesheets/govuk-component/_govspeak-html-publication-print.scss

```diff
@@ -1,5 +1,3 @@
-.govuk-govspeak-html-publication {
-  .govuk-sticky-element {
-    display: none;
-  }
+.govuk-sticky-element {
```

Comment:
This feels dangerous because now it's in the govspeak-html-publication-print.scss file but is a global selector. And it's not guaranteed we'll want to always hide anything using the sticky element feature.